src/_data/seo.yml

```
@@ -1,6 +1,6 @@
-favicon: /assets/images/logos/favicon.png
+favicon: /assets/uploads/nuxtjs.svg
 title: Primera Mesa
```

```suggestion
title: Venture
```
Cheeky perhaps but should we update this while we are here ?